autocertkit/utils.py

```diff
@@ -1932,6 +1932,29 @@ def xml_to_dicts(xml, tag):
     log.debug(result)
     return result
 
+def json_loads(json_data):
+    def process_dict_keys(d):
+        new_d = {}
+        for key in d.iterkeys():
+            new_key = str(key.replace(" ","_"))
```

Comment:
This change is to prevent regression. 
to_xml() method in ack plugin used to convert all whitespaces to _ so that xml tag will not have whitespace in name. ACK kit is written accordingly.

Json has no such limitation though using whitespace in key means changing ACK at many places where it is referring to key .
For example "Kernel name" -> "Kernel_name" and now ACK uses it at many places.

``` bash
auto-cert-kit/autocertkit# grep -r "Kernel_name"
utils.py:    ifaces = [dev['Kernel_name'] for dev in devices if device['PCI_id'] == dev['PCI_id']]
utils.py:        ifaces.append(device['Kernel_name'])
test_generators.py:            if device_rec['Kernel_name'] == self.interface:
ack_cli.py:                      if dev['Kernel_name'] in ifaces_to_test]
ack_cli.py:        ifaces.append(dev['Kernel_name'])
testbase.py:        log.debug("Equivalent devices for %s: %s" % (self.config['device_config']['Kernel_name'],
```